test/src/memory-level-test.js

```diff
@@ -1,5 +1,5 @@
-const memdown = require('memdown')
-const si = require('../../')
+const { MemoryLevel } = require('memory-level')
+const si = require('../../src/node')
```

Comment:
@mikaelkaron I think that this line is breaking the build and that it should be:

```javascript
const si = require('../../')
```

Other that it all looks good (as far as I can see)

Reply:
(line 2)

Reply:
clumsy me, that was not intended. fix pushed